tests/python/gaia-ui-tests/gaiatest/apps/search/app.py

```diff
@@ -33,15 +40,9 @@ def go_to_url(self, url):
 
     @property
     def history_items_count(self):
+        Wait(self.marionette).until(expected.element_present(*self._history_locator))
         return len(self.marionette.find_elements(*self._history_item_locator))
 
     def wait_for_history_to_load(self, number_of_items=1):
+        Wait(self.marionette).until(expected.element_present(*self._history_locator))
         Wait(self.marionette).until(lambda m: len(m.find_elements(*self._history_item_locator)) == number_of_items)
```

Comment:
Is this wait necessary? At a first glance, it seems redundant with the next line.